test-network/docker/docker-compose-test-net.yaml

```diff
@@ -76,6 +79,7 @@ services:
       - CORE_PEER_LOCALMSPID=Org1MSP
     volumes:
         - /var/run/:/host/var/run/
+        - ../../config/core.yaml:/etc/hyperledger/fabric/core.yaml
```

Comment:
if you mount the same ../../config/core.yaml file, to both org1 & org2 peer, will it work?

```
../../config/core.yaml file has a id as jdoe & listenAddress at port 7051

peer:
    id: jdoe
    listenAddress: 0.0.0.0:7051

```

Reply:
Hi Sijo, The important fields would be overridden by environment variables, which are different between the two orgs. The config files in the two peers are the same for both orgs in the current deployment as well.